### DISCUSSION TITLE
去年付的年费，升级7.8.4后至今还是免费版，请解决，谢谢

### DISCUSSION THREAD

**Discussion by Eddiekazsh** [getlantern/lantern-forum-cn#505] [General]
下方是我的付费收据。升级到7.8.4后就只能当免费版用。请协助解决，谢谢
![蓝灯收据](https://github.com/getlantern/lantern-forum-cn/assets/166973291/7aecb585-994a-41ac-8f46-f47c12b8304d)

**Comment by MaxMKMax** [comment]
@Eddiekazsh 您好. 请问您的账号邮箱 soflash@qq.com 是吗? 谢谢.

**Reply by Eddiekazsh** [reply]
对的，就是这个邮箱。@Mike777Khoo

**Reply by Eddiekazsh** [reply]
@Mike777Khoo 请尽快解决，前后都一个多月了

**Comment by MaxMKMax** [comment]
@Eddiekazsh 您好. 我已经帮您提交问题上去了. 这两天会给您发激活码. 请查看邮箱或Github. 谢谢!

**Reply by Eddiekazsh** [reply]
好的，非常感谢！

**Comment by MaxMKMax** [comment]
@Eddiekazsh 您的登入激活码:

C9RK2-CB8HJ-RC6B6-JFTVH-XJHCD

**Reply by Eddiekazsh** [reply]
收到，激活成功。谢谢！

**Reply by Eddiekazsh** [reply]
呃，我的激活码和邮箱都被你发出来了，不会被别人看到使用吧 @Mike777Khoo

**Reply by MaxMKMax** [reply]
@Eddiekazsh 被别人看到不要紧的. 这是您专用的. 只能一次性使用. 

不客气呢, 谢谢您支持蓝灯. 非常开心有像您这么尊贵的用户!

**Comment by Eddiekazsh** [comment]
输入激活码后，每次打开蓝灯还都是免费版，每次都需要输入。这么用了几天后。再输入激活码显示“兑换错误”，这是什么问题？ @Mike777Khoo

**Reply by MaxMKMax** [reply]
@Eddiekazsh 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
